### PR TITLE
Feature/cleanup and fixes

### PR DESCRIPTION
**Pull request by AD9018**
## Description

This fix addresses the fact that we need to separate by comma and space making it more intuitive rather than giving the user an alert that they have to use no space after the comma. It should be that they can type regularly meaning having a space after the comma. It is more semantic and user-friendly. Before it was rigid and the user could only search multiple values if they typed input right after the comma with no space. We also cleaned up code as well or useless functions. Functionality stretches to case table as well not just judge table. 

Co-Authored-by: Ricky Klusmeier

Fixes # (issue)
Cleanup of code as well as the implementation of a better way to search for multiple inputs. 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes
https://www.loom.com/share/28f903cc344f4aa79a59a201d80eeaf0

### CODE REVIEW

**Comment by lucaschatham** [comment]
You made the mistake I'm guilty of: forgot to add the Trello ticket. Other than that, the work looks solid. Nice work.

**Review by lucaschatham — APPROVED** [review]
You did what I'm guilty of: forgot to add the Trello ticket. Other than that, the work looks solid. Nice work.

**Review by willmoon19 — APPROVED** [review]
The clean-up looks good! I think It makes the code look a little nicer, so good job on that!
The spaces definitely make all the difference here, good catch! Keep it up!

**Review by ctscofield — APPROVED** [review]
Good catch to the search functionality! I like that you can now put a space after the comma before starting a new search, it makes it look a lot cleaner. And good job reviewing the code and cleaning out old commented out lines.

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -80,10 +69,10 @@ export default function CaseTable(props) {
     outcome: '',
     country_of_origin: '',
     gender: '',
-    type_of_persecution: '',
+    type_of_violence: '',
```
Comment (joshuascan) [inline src/components/pages/Cases/CaseTable.js:72]: Updated version is type_of_persecution

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -80,10 +69,10 @@ export default function CaseTable(props) {
...
     indigenous_group: '',
     applicant_language: '',
-    credibility: '',
+    credible: '',
```
Comment (joshuascan) [inline src/components/pages/Cases/CaseTable.js:75]: Updated version is credibility

**Review by joshuascan — CHANGES_REQUESTED** [review]
This looks good as well, except I have the same concern as the previous pull request where two of the variable names (see comments) have been changed back to their old version. I'm not sure if this happened in the first of three pull requests, or maybe it was a pull request prior to yours, but it's definitely important to get the schema realigned before this is merged.

**Review by prodhc — APPROVED** [review]
Nice job regarding the clean-up. These last 3 PRs you've put out are really important so it's great to see it all coming together. Again, can't find any different new issues that haven't been pointed out already.

**Review by RickyKlusmeier — APPROVED** [review]

**Review by joshuascan — APPROVED** [review]

**Review by matthewblackbu — APPROVED** [review]
Looks good the formatting looks a lit better and the functionality is there just make sure the persecution thing is looked into

**Pull request merged**